src/Model/Notify.php

```diff
@@ -0,0 +1,775 @@
+	 */
+	public function getSystemNotifies(bool $seen = false, int $start = 0, int $limit = self::DEFAULT_PAGE_LIMIT)
+	{
+		$ident    = self::SYSTEM;
+		$notifies = [];
+
+		$filter = ['uid' => local_user()];
+		if (!$seen) {
+			$filter['seen'] = false;
+		}
+
+		$params          = [];
+		$params['order'] = ['date' => 'DESC'];
+		$params['limit'] = [$start, $limit];
+
+		$stmtNotifies = $this->dba->select('notify',
+			['id', 'url', 'photo', 'msg', 'date', 'seen', 'verb'],
+			$filter,
+			$params);
+
+		if ($this->dba->isResult($stmtNotifies)) {
+			$notifies = $this->formatNotifies($this->dba->toArray($stmtNotifies), $ident);
+		}
+
+		$arr = [
+			'notifications' => $notifies,
+			'ident'         => $ident,
+		];
+
+		return $arr;
+	}
+
+	/**
+	 * Get personal notifications
+	 *
+	 * @param bool $seen    False => only include notifications into the query
+	 *                            which aren't marked as "seen"
+	 * @param int        $start   Start the query at this point
+	 * @param int        $limit   Maximum number of query results
+	 *
+	 * @return array [string, array]
+	 *    string 'ident' => Notification identifier
+	 *    array 'notifications' => Personal notifications
+	 *
+	 * @throws Exception
+	 */
+	public function getPersonalNotifies(bool $seen = false, int $start = 0, int $limit = self::DEFAULT_PAGE_LIMIT)
+	{
+		$ident    = self::PERSONAL;
+		$notifies = [];
+
+		$myurl     = str_replace('http://', '', self::getApp()->contact['nurl']);
+		$diasp_url = str_replace('/profile/', '/u/', $myurl);
+
+		$condition = ["NOT `wall` AND `uid` = ? AND (`item`.`author-id` = ? OR `item`.`tag` REGEXP ? OR `item`.`tag` REGEXP ?)",
+			local_user(), public_contact(), $myurl . '\\]', $diasp_url . '\\]'];
+
+		if (!$seen) {
+			$condition[0] .= " AND `unseen`";
+		}
+
+		$fields = ['id', 'parent', 'verb', 'author-name', 'unseen', 'author-link', 'author-avatar', 'contact-avatar',
+			'network', 'created', 'object', 'parent-author-name', 'parent-author-link', 'parent-guid'];
+		$params = ['order' => ['received' => true], 'limit' => [$start, $limit]];
+
+		$items = Item::selectForUser(local_user(), $fields, $condition, $params);
+
+		if ($this->dba->isResult($items)) {
+			$notifies = $this->formatNotifies(Item::inArray($items), $ident);
+		}
+
+		$arr = [
+			'notifications' => $notifies,
+			'ident'         => $ident,
+		];
+
+		return $arr;
+	}
+
+	/**
+	 * @brief Get home notifications
+	 *
+	 * @param bool $seen    False => only include notifications into the query
+	 *                            which aren't marked as "seen"
+	 * @param int        $start   Start the query at this point
+	 * @param int        $limit   Maximum number of query results
+	 *
+	 * @return array [string, array]
+	 *    string 'ident' => Notification identifier
+	 *    array 'notifications' => Home notifications
+	 *
+	 * @throws Exception
+	 */
+	public function getHomeNotifies(bool $seen = false, int $start = 0, int $limit = self::DEFAULT_PAGE_LIMIT)
```

Comment:
`Notify` already is the name of the class, this should be renamed `getHomeList`.